References.md

```diff
@@ -1,24 +1,25 @@
--https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference
--firebase documentation/developer information
--https://davidwalsh.name/nodejs-http-request
--https://nodejs.org/api/http.html
--https://devcenter.heroku.com/articles/ssl-certificate-self
--expressjs documentation
--firebase documentation
--nodejs documentation
--MDN js documentation
--npm documentation
--https://lord.io/blog/2014/travis-multiple-subdirs/
+- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference
```

Comment:
Can References.md be moved to the wiki, or do you prefer keeping them with our code?